devtronic/storybook
# Fixed broken button example story for Angular

Issue:
The button example story in Angular is broken. The text isn't displayed since the property name was renamed in 24ec9d3674fe8a2b151144803c1443c6d0a389ab.

## What I did
Made the property name lowercase.

## How to test

- Is this testable with Jest or Chromatic screenshots? Idk.
- Does this need a new example in the kitchen sink apps? Idk.
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
